demo/src/demoApp.js

```diff
@@ -41,12 +41,26 @@ import {registerScreens} from './screens';
 //   camera: require('./assets/icons/cameraSelected.png'),
 // });
 
-Animatable.initializeRegistryWithDefinitions({
-  basicListEntrance: {
-    from: {opacity: 0, translateY: 20},
-    to: {opacity: 1, translateY: 0},
-  },
-});
+// Animatable.initializeRegistryWithDefinitions({
+//   basicListEntrance: {
+//     from: {opacity: 0, translateY: 20},
+//     to: {opacity: 1, translateY: 0},
+//   },
+// });
+
+// const costumDefinitions = {
+//   costumAnimation1: {
+//     from: {opacity: 0, translateY: 20},
+//     to: {opacity: 1, translateY: 0},
+//   }, 
+//   costumAnimation2: {
+//     from: {opacity: 0, translateY: 40},
+//     to: {opacity: 1, translateY: 0},
+//   },
+// };
+
+Animatable.initializeRegistryWithDefinitions(AnimatableManager.loadCustomDefinitions(/** costumDefinitions */));
```

Comment:
The uilib animation presets should already be part of uilib when loaded, exactly like colors, typography and other presets.

Reply:
Done.